packages/client/src/open-feature.ts

```diff
@@ -75,6 +75,10 @@ export class OpenFeatureAPI extends OpenFeatureCommonAPI<Provider> implements Ma
       { name, version }
     );
   }
+
+  clearProviders() {
```

Comment:
JSDocs are enhanced from the abstract class.

Reply:
Are they even though abstract and concrete `clearProviders` they have different params?

Reply:
Hmm, good point. Let me double-check.

Reply:
I've added a JSDoc to both the client and server because the method name on the abstract class has changed. That should resolve this concern and the LSP violation.